Systems ref jcayouette (Content Lifecycle Update) (#433)
SUSE/doc-susemanager: modules/reference/pages/configuration/config-systems-target.adoc

[[config-systems-target]]
= Target Systems

> Here you see the systems either not prepared for configuration file deployment or not yet subscribed to a configuration channel.

Reviewer: ```suggestion
This page shows the systems that are not prepared for configuration file deployment, or are not yet subscribed to a configuration channel.
```

> The table has three columns.

Reviewer: ```suggestion
The table has three columns:
```
